src/main/features/claimant-response/routes/claimant-hearing-receipt.ts

```diff
@@ -0,0 +1,21 @@
+import * as express from 'express'
+import { Paths } from 'claimant-response/paths'
+
+import { Claim } from 'claims/models/claim'
+import { DocumentsClient } from 'documents/documentsClient'
+
+import { ErrorHandling } from 'shared/errorHandling'
+
+import { DownloadUtils } from 'utils/downloadUtils'
+
+const documentsClient: DocumentsClient = new DocumentsClient()
+
+/* tslint:disable:no-default-export */
+export default express.Router()
+  .get(Paths.claimantReceiptReceiver.uri,
+    ErrorHandling.apply(async (req: express.Request, res: express.Response): Promise<void> => {
+      const claim: Claim = res.locals.claim
+
+      const pdf: Buffer = await documentsClient.getClaimantHearingRequirementPDF(claim.externalId, res.locals.user.bearerToken)
+      DownloadUtils.downloadPDF(res, pdf, `${claim.claimNumber}-claim-form-claimant-copy`)
+    }))
```

Comment:
Do we need test for this? not sure, just asking!

Reply:
Done. Thanks for the help @KiranVarma02